_translator-files/po/fr/Contribution.po

```diff
@@ -59,10 +59,14 @@ msgstr "Si vous pensez qu'il manque de la documentation ou des informations ou q
 
 #. type: Plain text
 #: ../wiki/en/Contribution.md:15
-#, fuzzy
-#| msgid "If you want to help out with translation, you will need to have a GitHub account and should be familiar with using git and Jekyll. Read about our [documentation and translation process](https://github.com/jamulussoftware/jamuluswebsite/blob/release/README.md), and [introduce yourself](https://github.com/jamulussoftware/jamulus/discussions) if you’d like to get involved. You can check the current status of website translations [here](Statistics)."
 msgid "We use [Weblate](https://hosted.weblate.org/projects/jamulus/#languages) to translate our documentation. Read about our [documentation and translation process](https://github.com/jamulussoftware/jamuluswebsite/blob/release/README.md), and [introduce yourself](https://github.com/jamulussoftware/jamulus/discussions) if you’d like to get involved."
-msgstr "Si vous souhaitez aider à la traduction, vous devrez disposer d'un compte GitHub et être familier avec l'utilisation de git et de Jekyll. Lisez notre [processus de documentation et de traduction](https://github.com/jamulussoftware/jamuluswebsite/blob/release/README.md), et [présentez-vous](https://github.com/jamulussoftware/jamulus/discussions) si vous souhaitez vous impliquer. Vous pouvez vérifier l'état actuel des traductions du site web [ici](Statistics)."
+msgstr ""
+"Nous utilisons [Weblate](https://hosted.weblate.org/projects/jamulus/"
+"#languages) pour traduire notre documentation. Lire à propos notre ["
```

Comment:
The line breaks here look prone to error.  Is it correct to insert a line break after a `[`?

Reply:
@trebmuh

Reply:
As far as I know, it's not a problem at all. It's not really "cut" and will be "re-glued" by whatever is processing the file after.
I mean, at least in a lot of other software I'm (or have been) translating, this is the same.